Configurations/unix-Makefile.tmpl

```diff
@@ -741,56 +741,56 @@ lint:
 	lint -DLINT $(INCLUDES) $(SRCS)
 
 generate_apps:
-	( cd $(SRCDIR); $(PERL) VMS/VMSify-conf.pl \
+	( cd "$(SRCDIR)"; $(PERL) VMS/VMSify-conf.pl \
 				< apps/openssl.cnf > apps/openssl-vms.cnf )
 	@ : {- output_off() if $disabled{apps}; "" -}
-	( b=`pwd`; cd $(SRCDIR); \
+	( b=`pwd`; cd "$(SRCDIR)"; \
 	  $(PERL) -I$$b apps/progs.pl -H $(APPS_OPENSSL) > apps/progs.h )
-	( b=`pwd`; cd $(SRCDIR); \
+	( b=`pwd`; cd "$(SRCDIR)"; \
 	  $(PERL) -I$$b apps/progs.pl -C $(APPS_OPENSSL) > apps/progs.c )
 	@ : {- output_on() if $disabled{apps}; "" -}
 
 generate_crypto_bn:
-	( cd $(SRCDIR); $(PERL) crypto/bn/bn_prime.pl > crypto/bn/bn_prime.h )
+	( cd "$(SRCDIR)"; $(PERL) crypto/bn/bn_prime.pl > crypto/bn/bn_prime.h )
 
 generate_crypto_objects:
-	( cd $(SRCDIR); $(PERL) crypto/objects/objects.pl -n \
+	( cd "$(SRCDIR)"; $(PERL) crypto/objects/objects.pl -n \
 				crypto/objects/objects.txt \
 				crypto/objects/obj_mac.num \
 				> crypto/objects/obj_mac.new && \
 	    mv crypto/objects/obj_mac.new crypto/objects/obj_mac.num )
-	( cd $(SRCDIR); $(PERL) crypto/objects/objects.pl \
+	( cd "$(SRCDIR)"; $(PERL) crypto/objects/objects.pl \
 				crypto/objects/objects.txt \
 				crypto/objects/obj_mac.num \
 				> include/openssl/obj_mac.h )
-	( cd $(SRCDIR); $(PERL) crypto/objects/obj_dat.pl \
+	( cd "$(SRCDIR)"; $(PERL) crypto/objects/obj_dat.pl \
 				include/openssl/obj_mac.h \
 				> crypto/objects/obj_dat.h )
-	( cd $(SRCDIR); $(PERL) crypto/objects/objxref.pl \
+	( cd "$(SRCDIR)"; $(PERL) crypto/objects/objxref.pl \
 				crypto/objects/obj_mac.num \
 				crypto/objects/obj_xref.txt \
 				> crypto/objects/obj_xref.h )
 
 generate_crypto_conf:
-	( cd $(SRCDIR); $(PERL) crypto/conf/keysets.pl \
+	( cd "$(SRCDIR)"; $(PERL) crypto/conf/keysets.pl \
 			        > crypto/conf/conf_def.h )
 
 generate_crypto_asn1:
-	( cd $(SRCDIR); $(PERL) crypto/asn1/charmap.pl \
+	( cd "$(SRCDIR)"; $(PERL) crypto/asn1/charmap.pl \
 			        > crypto/asn1/charmap.h )
 
 generate_fuzz_oids:
-	( cd $(SRCDIR); $(PERL) fuzz/mkfuzzoids.pl \
+	( cd "$(SRCDIR)"; $(PERL) fuzz/mkfuzzoids.pl \
 				crypto/objects/obj_dat.h \
 				> fuzz/oids.txt )
 
 # Set to -force to force a rebuild
 ERROR_REBUILD=
 errors:
-	( b=`pwd`; set -e; cd $(SRCDIR); \
+	( b=`pwd`; set -e; cd "$(SRCDIR)"; \
           $(PERL) util/ck_errf.pl -strict -internal; \
           $(PERL) -I$$b util/mkerr.pl $(ERROR_REBUILD) -internal )
```

Comment:
quote -I$$b ?

Reply:
Yes, and also `pwd`.

Any reason why this is used, instead of `find [...] -delete`:
```shell
$(RM) `find . -name '*{- platform->depext() -}' \! -name '.*' -print`
```

Reply:
Not sure I understand your question here

Reply:
What I mean is, `find` in this form is unsafe, because filenames containing spaces will be split into pieces and passed as such to `rm`.

Using the form below would work as intended for _any_ filename, even if they include spaces:
```shell
find . -name '*{- platform->depext() -}' \! -name '.*' -delete
```

Reply:
no issue since find start with . and we do not create spaces below the build dir.